xbmc/PlayListPlayer.h

```diff
@@ -11,8 +11,10 @@
 #include "ServiceBroker.h"
 #include "guilib/IMsgTargetCallback.h"
 #include "messaging/IMessageTarget.h"
+#include "playlists/PlayList.h"
```

Comment:
```suggestion
#include "playlists/PlayListTypes.h"
```
If not in any files that includes `PlayListPlayer.h` also includes BIG `playlists/PlayList.h`